docs/UsersGuide/source/BuildingRunningTesting/BuildSRW.rst

```diff
@@ -77,7 +79,9 @@ The cloned repository contains the configuration files and sub-directories shown
 
 .. code-block:: console
 
+    # In a bash shell, run:
     export SRW=$HOME/ufs-srweather-app
+    # In a csh shell, run: 
```

Comment:
You and Christina think alike. ;) I'm not opposed, but I'm curious why? From the perspective of streamlining the documentation instead of listing variants of the same commands, I agree. From the perspective of portability, it seems good to keep csh. When I started, I would've had no idea what a shell was or how to figure it out, so if your system default isn't bash, that could be confusing. But then, if most systems use bash, a note for the outliers might be sufficient. Also, I think there are more shells than bash and csh, so if we have csh, people could reasonably ask why we don't support others, too? (maintainability nightmare much?) There you have my rambling thoughts on the matter. :)

Reply:
Mainly for the reasons in the linked article above, it's just an inferior shell for most if not all use cases. Like you allude to, trying to support multiple shells leads to clunky documentation, so if we're going to settle on one it should definitely be bash. Plus, all of our workflow scripting is in bash (and it is the language of crontabs as well), so it's thematically appropriate.

Reply:
Oops, overlooked the link, sorry! But yes. I think this is a good move. I think we should probably run it by @MichaelLueken before implementing tho. Since I've got 2 more PRs coming in, I think I could just add the changes to one of them.